benchmark/cmake/modules/AddSwiftBenchmarkSuite.cmake

```diff
@@ -323,6 +332,17 @@ function (swift_benchmark_compile_archopts)
             "-o" "${sibfile}"
             "${source}")
       endif()
+
+      if("${common_options}" MATCHES "save-optimization-record")
+        set(opt_record "${objdir}/${module_name}.opt.yaml")
+        set(opt_viewer "${SWIFT_PATH_TO_LLVM_SOURCE}/tools/opt-viewer/opt-viewer.py")
+        set(opt_view_dir "${optview_main_dir}/${module_name}")
+        add_custom_command(
```

Comment:
More out of curiosity than anything else, will optview_main_dir exist?

Reply:
Yes, opt-viewer creates the necessary directories.

Reply:
k